onert-micro/onert-micro/include/test_models/zeroslike/FloatZerosLikeKernel.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
```suggestion
 * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
```

Reply:
The majority of the code comes from the legacy luci_interpreter's. how about keeping it ?